tcl/tools/analysis.tcl

```diff
@@ -897,14 +898,14 @@ proc configAnnotation {n} {
 
     ttk::labelframe $f.analyse -text $::tr(GameReview)
     ttk::label $f.analyse.label -text $::tr(AnnotateTime)
-    ttk::spinbox $f.analyse.spDelay -background white -width 4 -textvariable tempdelay -from 1 -to 999 -increment 1 \
-        -validate key -validatecommand { return [string is digit %S] }
+    ttk::spinbox $f.analyse.spDelay -width 5 -textvariable tempdelay -from 0.5 -to 999 -increment 0.1 \
```

Comment:
This doesn't seem to work.
The validate command accept only digits, so it is only possible to enter integer numbers.
Also incrementing by 0.1 is too slow.
I think the increment should be left to 1, but the validate command should be changed to accept decimal numbers.